src/abstract-tracker/abstract-tracker.component.ts

```diff
@@ -28,7 +28,7 @@ export abstract class AbstractTrackerComponent {
    * RELATED ISSUE: https://github.com/angular/angular/issues/4402
    * TODO: Create a base component class that implements track by for all
    */
-  trackByFunction(index: number, obj: any): any {
-    return index;
+  trackByFunction(index: number, element: any): any {
```

Comment:
btw, do we still need the comment above? The issue https://github.com/angular/angular/issues/4402 is closed.